scripts/genpinctrl/stm32-pinctrl-config.yaml

```diff
@@ -267,3 +267,6 @@
 
 - name: USB
   match: "^USB_(?:DM)?(?:DP)?(?:NOE)?$"
+
+- name: JTAG PORT
+  match: ^(SYS|DEBUG)_(JTMS-SWDIO|JTCK-SWCLK|JTDI|JTDO(-TRACESWO|-SWO)?|(NJ)?JTRST)$
```

Comment:
On C0, the debug pins are "DEBUG_SWDIO" and "DEBUG_SWCLK".
On F0/G0/L0, they are "SYS_SWDIO" and "SYS_SWCLK".
Is it useful to add these patterns too?
Would the string
`^(SYS|DEBUG)_((JTMS-)?SWDIO|(JTCK-)?SWCLK|JTDI|JTDO(-TRACESWO|-SWO)?|(NJ)?JTRST)$`
work for these?

Reply:
Thanks, I'll have a check